### PR TITLE
feat: add `editorial_summary` to atmosphere fields

### LINKED ISSUE #471: Add `editorial_summary` to place atmosphere field dictionary
As part of a recent [feature request](https://issuetracker.google.com/issues/35827225) that Google completed, a field labeled **`editorial_summary`** was as added to the response to Place Details API endpoint (`place()` in this package). 

To allow Python users to access this field via this API package, please add **`editorial_summary`** to the **PLACES_DETAIL_FIELDS_ATMOSPHERE** [dictionary](https://github.com/googlemaps/google-maps-services-python/blob/master/googlemaps/places.py#L83) and **PLACES_FIND_FIELDS_ATMOSPHERE** [dictionary](https://github.com/googlemaps/google-maps-services-python/blob/master/googlemaps/places.py#L47). As of now, if `editorial_summary` is listed in the requested fields in the `place()` function, a ValueError is returned. 

This editorial summary allows users to access the Google Business description for each place, which is helpful to gain more fine grained descriptions about places beyond the standard classification provided by Google.

### PR DESCRIPTION
**Pull request by jloffredo2**
Fixes #471 🦕

### CODE REVIEW

**Comment by jloffredo2** [comment]
CLA has been signed

File: googlemaps/places.py [hunk 1]
```diff
@@ -44,7 +44,7 @@
 
 PLACES_FIND_FIELDS_CONTACT = {"opening_hours"}
 
-PLACES_FIND_FIELDS_ATMOSPHERE = {"price_level", "rating", "user_ratings_total"}
+PLACES_FIND_FIELDS_ATMOSPHERE = {"editorial_summary","price_level", "rating", "user_ratings_total"}
```
Comment (wangela) [inline googlemaps/places.py:47]: `editorial_summary` is not supported for Find Place. See [documentation](https://developers.google.com/maps/documentation/places/web-service/place-data-fields#places-api-fields-support).

**Review by wangela — CHANGES_REQUESTED** [review]
Thank you for your pull request. Please revert the change to `PLACES_FIND_FIELDS_ATMOSPHERE` and we'll be good to go. 

Noting here that it would be inadvisable to add this field to `tests/test_places.py` since the editorial summary content might change.

**Comment by jloffredo2** [comment]
@wangela As per your request, I have reverted the change to `PLACES_FIND_FIELDS_ATMOSPHERE` and have added `feat:` to this PR title.

**Review by wangela — APPROVED** [review]
Thanks for incorporating the feedback!

**Pull request merged**